src/network-validation/network-validation.service.ts

```diff
@@ -0,0 +1,84 @@
+import { MikroORM, UseRequestContext } from '@mikro-orm/core';
```

Comment:
lets write tests on this service ?

Reply:
OK, good idea. Will do.

Reply:
I implemented unit tests for the `NetworkValidation` service.